crates/red_knot_python_semantic/src/types/infer.rs

```diff
@@ -55,12 +55,14 @@ use crate::types::diagnostic::{
 use crate::types::{
     bindings_ty, builtins_symbol_ty, declarations_ty, global_symbol_ty, symbol_ty,
     typing_extensions_symbol_ty, BytesLiteralType, ClassType, FunctionType, IterationOutcome,
-    KnownClass, KnownFunction, SliceLiteralType, StringLiteralType, Truthiness, TupleType, Type,
-    TypeArrayDisplay, UnionBuilder, UnionType,
+    KnownClass, KnownFunction, SliceLiteralType, StringLiteralType, SymbolLookupResult, Truthiness,
+    TupleType, Type, TypeArrayDisplay, UnionBuilder, UnionType,
 };
 use crate::util::subscript::{PyIndex, PySlice};
 use crate::Db;
 
+use super::Boundedness;
```

Comment:
Yes. I also configured rust-analyzer to do this correctly/consistently in the future by setting `imports.prefix` to `crate` instead of `plain`: https://rust-analyzer.github.io/manual.html#auto-import